package.json

```diff
@@ -49,8 +49,8 @@
   },
   "homepage": "https://github.com/checkdigit/eslint-config#readme",
   "devDependencies": {
-    "@checkdigit/prettier-config": "^5.4.0",
-    "@checkdigit/typescript-config": "^7.0.1",
+    "@checkdigit/prettier-config": "^5.4.1",
+    "@checkdigit/typescript-config": "7.1.0-PR.58-2978",
```

Comment:
Requires https://github.com/checkdigit/typescript-config/pull/58 to be squerged first